docs/azure-devops-wiki.md

```diff
@@ -31,4 +31,39 @@ There is default table of contents pane to the left of the wiki content. Here, y
 
 _Note that first entry in `.order` file defines wiki's home page._
 
+## Update wiki
+
+While you are pushing changes to GitHub, you won't see the wiki updated if you don't add Azure as a remote. You can push to multiple repositories simultaneously.
+ 
+ 1. First check if Azure is added: `git remote show origin`. If not add it following these steps.
+ 2. Rename your current remote (most likely named origin) to a different name.
```

Comment:
How do you do this? I think it would be helpful to tell them and give a suggested name.

Reply:
Apologies, I don't think GitHub made it clear. I meant on 2. Can we tell them how to rename their current remote?

Reply:
Thanks for your change.